profiling/profiling.go

```diff
@@ -0,0 +1,20 @@
+package profiling
+
+import (
+	"net"
+	"net/http"
+
+	// Required for profiling
+	_ "net/http/pprof"
+)
+
+// Start starts the profiling server
+func Start(port string) {
+	spawn(func() {
+		listenAddr := net.JoinHostPort("", port)
+		log.Infof("Profile server listening on %s", listenAddr)
+		profileRedirect := http.RedirectHandler("/debug/pprof", http.StatusSeeOther)
+		http.Handle("/", profileRedirect)
+		log.Errorf("%s", http.ListenAndServe(listenAddr, nil))
```

Comment:
Why do you need Errorf here? Just use log.Error(), or wrap with some error message.